<ui version="4.0" >
 <class>SensorsCnfgDialog</class>
 <widget class="QDialog" name="SensorsCnfgDialog" >
  <property name="windowModality" >
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>566</width>
    <height>358</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Sensor Configuration</string>
  </property>
  <widget class="QGroupBox" name="uiGrp1stCardSensors" >
   <property name="geometry" >
    <rect>
     <x>20</x>
     <y>10</y>
     <width>411</width>
     <height>141</height>
    </rect>
   </property>
   <property name="title" >
    <string>Sensors on 1st Card:</string>
   </property>
   <widget class="QComboBox" name="uiComboSensor1Trigger" >
    <property name="geometry" >
     <rect>
      <x>66</x>
      <y>20</y>
      <width>104</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QComboBox" name="uiComboSensor2Trigger" >
    <property name="geometry" >
     <rect>
      <x>66</x>
      <y>50</y>
      <width>104</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QComboBox" name="uiComboSensor3Trigger" >
    <property name="geometry" >
     <rect>
      <x>66</x>
      <y>80</y>
      <width>104</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QComboBox" name="uiComboSensor4Trigger" >
    <property name="geometry" >
     <rect>
      <x>66</x>
      <y>110</y>
      <width>104</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QComboBox" name="uiComboSensor5Trigger" >
    <property name="geometry" >
     <rect>
      <x>286</x>
      <y>20</y>
      <width>104</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QComboBox" name="uiComboSensor6Trigger" >
    <property name="geometry" >
     <rect>
      <x>286</x>
      <y>50</y>
      <width>104</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QComboBox" name="uiComboSensor7Trigger" >
    <property name="geometry" >
     <rect>
      <x>286</x>
      <y>80</y>
      <width>104</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QComboBox" name="uiComboSensor8Trigger" >
    <property name="geometry" >
     <rect>
      <x>286</x>
      <y>110</y>
      <width>104</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QCheckBox" name="uiChkEnableSensor1" >
    <property name="geometry" >
     <rect>
      <x>20</x>
      <y>20</y>
      <width>40</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text" >
     <string>1</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="uiChkEnableSensor2" >
    <property name="geometry" >
     <rect>
      <x>20</x>
      <y>50</y>
      <width>39</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text" >
     <string>2</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="uiChkEnableSensor3" >
    <property name="geometry" >
     <rect>
      <x>20</x>
      <y>80</y>
      <width>40</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text" >
     <string>3</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="uiChkEnableSensor4" >
    <property name="geometry" >
     <rect>
      <x>20</x>
      <y>110</y>
      <width>40</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text" >
     <string>4</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="uiChkEnableSensor5" >
    <property name="geometry" >
     <rect>
      <x>240</x>
      <y>20</y>
      <width>39</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text" >
     <string>5</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="uiChkEnableSensor6" >
    <property name="geometry" >
     <rect>
      <x>240</x>
      <y>50</y>
      <width>40</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text" >
     <string>6</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="uiChkEnableSensor7" >
    <property name="geometry" >
     <rect>
      <x>240</x>
      <y>80</y>
      <width>39</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text" >
     <string>7</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="uiChkEnableSensor8" >
    <property name="geometry" >
     <rect>
      <x>240</x>
      <y>110</y>
      <width>40</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text" >
     <string>8</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="uiGrp2ndCardSensors" >
   <property name="geometry" >
    <rect>
     <x>20</x>
     <y>160</y>
     <width>411</width>
     <height>141</height>
    </rect>
   </property>
   <property name="title" >
    <string>Sensors on 2nd Card:</string>
   </property>
   <widget class="QCheckBox" name="uiChkEnableSensor09" >
    <property name="geometry" >
     <rect>
      <x>20</x>
      <y>20</y>
      <width>40</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text" >
     <string>09</string>
    </property>
   </widget>
   <widget class="QComboBox" name="uiComboSensor09Trigger" >
    <property name="geometry" >
     <rect>
      <x>66</x>
      <y>20</y>
      <width>104</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QCheckBox" name="uiChkEnableSensor10" >
    <property name="geometry" >
     <rect>
      <x>20</x>
      <y>50</y>
      <width>39</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text" >
     <string>10</string>
    </property>
   </widget>
   <widget class="QComboBox" name="uiComboSensor10Trigger" >
    <property name="geometry" >
     <rect>
      <x>66</x>
      <y>50</y>
      <width>104</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QCheckBox" name="uiChkEnableSensor11" >
    <property name="geometry" >
     <rect>
      <x>20</x>
      <y>80</y>
      <width>40</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text" >
     <string>11</string>
    </property>
   </widget>
   <widget class="QComboBox" name="uiComboSensor11Trigger" >
    <property name="geometry" >
     <rect>
      <x>66</x>
      <y>80</y>
      <width>104</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QCheckBox" name="uiChkEnableSensor12" >
    <property name="geometry" >
     <rect>
      <x>20</x>
      <y>110</y>
      <width>40</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text" >
     <string>12</string>
    </property>
   </widget>
   <widget class="QComboBox" name="uiComboSensor12Trigger" >
    <property name="geometry" >
     <rect>
      <x>66</x>
      <y>110</y>
      <width>104</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QCheckBox" name="uiChkEnableSensor13" >
    <property name="geometry" >
     <rect>
      <x>240</x>
      <y>20</y>
      <width>39</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text" >
     <string>13</string>
    </property>
   </widget>
   <widget class="QComboBox" name="uiComboSensor13Trigger" >
    <property name="geometry" >
     <rect>
      <x>286</x>
      <y>20</y>
      <width>104</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QCheckBox" name="uiChkEnableSensor14" >
    <property name="geometry" >
     <rect>
      <x>240</x>
      <y>50</y>
      <width>40</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text" >
     <string>14</string>
    </property>
   </widget>
   <widget class="QComboBox" name="uiComboSensor14Trigger" >
    <property name="geometry" >
     <rect>
      <x>286</x>
      <y>50</y>
      <width>104</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QComboBox" name="uiComboSensor15Trigger" >
    <property name="geometry" >
     <rect>
      <x>286</x>
      <y>80</y>
      <width>104</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QCheckBox" name="uiChkEnableSensor15" >
    <property name="geometry" >
     <rect>
      <x>240</x>
      <y>80</y>
      <width>39</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text" >
     <string>15</string>
    </property>
   </widget>
   <widget class="QComboBox" name="uiComboSensor16Trigger" >
    <property name="geometry" >
     <rect>
      <x>286</x>
      <y>110</y>
      <width>104</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QCheckBox" name="uiChkEnableSensor16" >
    <property name="geometry" >
     <rect>
      <x>240</x>
      <y>110</y>
      <width>40</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text" >
     <string>16</string>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="uiBtnEdgeTriggerAll" >
   <property name="geometry" >
    <rect>
     <x>440</x>
     <y>110</y>
     <width>116</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text" >
    <string>Edge Trigger All</string>
   </property>
  </widget>
  <widget class="QPushButton" name="uiBtnEnableAll" >
   <property name="geometry" >
    <rect>
     <x>440</x>
     <y>20</y>
     <width>116</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text" >
    <string>Enable All</string>
   </property>
  </widget>
  <widget class="QPushButton" name="uiBtnDisableAll" >
   <property name="geometry" >
    <rect>
     <x>440</x>
     <y>50</y>
     <width>116</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text" >
    <string>Disable All</string>
   </property>
  </widget>
  <widget class="QPushButton" name="uiBtnLevelSensitiveAll" >
   <property name="geometry" >
    <rect>
     <x>440</x>
     <y>141</y>
     <width>116</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text" >
    <string>Level Sensitive All</string>
   </property>
  </widget>
  <widget class="QDialogButtonBox" name="buttonBox" >
   <property name="geometry" >
    <rect>
     <x>380</x>
     <y>310</y>
     <width>180</width>
     <height>41</height>
    </rect>
   </property>
   <property name="orientation" >
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons" >
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
   <property name="centerButtons" >
    <bool>true</bool>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>SensorsCnfgDialog</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>288</x>
     <y>392</y>
    </hint>
    <hint type="destinationlabel" >
     <x>277</x>
     <y>174</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>SensorsCnfgDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>319</x>
     <y>380</y>
    </hint>
    <hint type="destinationlabel" >
     <x>317</x>
     <y>400</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
